### PR TITLE
Seven forum fixes: reply button direction and markdown editor buttons, community TA label, font-size, image upload, callout image and upload image

### PR DESCRIPTION
**Pull request by OmarIthawi**
Tasks: 

 - [content-discussion/Arabic interface: The Scroll bar should be displayed to the right moreover the "تقديم" button under the comment field should be aligned to the right](https://app.asana.com/0/96288420553298/140074164445564)
 - [Discussion: UI Issues](https://app.asana.com/0/96288420553298/140259360226853)
 - [Content-specific discussion topic: the discussion topic font size is too large moreover the design is different from the production](https://app.asana.com/0/96288420553298/139973942243266)
 - [Discussion/Image upload: The "Maximum upload size...." validation message is displayed when uploading two images Sequentially ( The first image size > 1 mb and the second one <1 mb)](https://app.asana.com/0/96288420553298/130892784707019)
 - [Discussion: Posts that you have read, but with responses or comments that you have not read yet, it should have a white callout image.](https://app.asana.com/0/96288420553298/140259360226846)
 - [Discussion/Image upload: The "Upload file" box displays improperly for "content specific discussion"](https://app.asana.com/0/96288420553298/130892784707031)

Related Theme PR:

 - [Remove the horizontal scrollbar from the forum reply button](https://bitbucket.org/Edraak/edraak-2015-theme/pull-requests/19/remove-the-horizontal-scrollbar-from-the/diff)
 - [Removed the discussion hacks, moved it to the  #platform](https://bitbucket.org/Edraak/edraak-2015-theme/pull-requests/20)

### CODE REVIEW

**Comment by ahmedaljazzar** [comment]
كل اللي في مطار فرانكفورت عرب. مش مصدق حالي!

**Comment by OmarIthawi** [comment]
ههههههه الحمد لله عالسلامة

File: lms/static/sass/discussion/views/_thread.scss [hunk 1]
```diff
@@ -34,6 +34,7 @@ body.discussion, .discussion-module {
     }
 
     .posted-details {
+      @extend %t-copy-sub2;  // Edraak: Corrected the font-size
```
Comment (ahmedaljazzar) [inline lms/static/sass/discussion/views/_thread.scss]: Too large, no? The production still looks gentler.

**Beta**
screen shot 2016-06-28 at 10 42 47 am

**Production**
screen shot 2016-06-28 at 10 42 40 am

**Comment by ahmedaljazzar** [comment]
This dialog had been shown while uploading an image larger than 1MB.

screen shot 2016-06-28 at 11 08 04 am

File: lms/static/sass/discussion/views/_thread.scss (hunk 1 above)
Comment (OmarIthawi) [inline lms/static/sass/discussion/views/_thread.scss]: edX decided so. I think this is a matter of opinion, and the easiest way is to pick what edX did.

**Comment by OmarIthawi** [comment]
> This dialog had been shown while uploading an image larger than 1MB.

QA?

**Comment by OmarIthawi** [comment]
@ahmedaljazzar I checked the issue. It happens only when you upload a too-large (larger than 4MB) file. For example, for example uploading 2MB file will be OK.

I double checked and it turns out that it's an edX bug as well.


- 1.3 MB Image: 
screen shot 2016-06-28 at 4 41 39 pm

- 7MB Image - Safari:  screen shot 2016-06-28 at 4 42 57 pm

- 7MB Image Chrome: screen shot 2016-06-28 at 4 43 49 pm

**Comment by OmarIthawi** [comment]
@ahmedaljazzar So this PR makes Edraak as good as https://edX.org .

This specific >4MB bug I suggest that it gets filed on another bug.

**Pull request merged**